ci/upload-console.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+source ci/upload-ci-artifact.sh
+
+set -e
+
+if [[ -n $CI ]]; then
+  exit_code=0
+  console_out="console-$(date '+%Yy%mm%dd%Hh%Mm%Ss%Nns').log"
+
+  # shellcheck disable=SC2094
```

Comment:
i know this is racy. but i don't think this is deal breaker. for one thing, original code is racy as well.